assets/js/state/selectors/lastExecutions.js

```diff
@@ -0,0 +1,3 @@
+export const getLastExecution = (groupID) => ({lastExecutions}) => {
+  return lastExecutions[groupID];
```

Comment:
What would happen here if we don't have this entry? Like we don't have any execution yet for this group. Would this break something?
It would return `undefined` and most probably break something in the clusters view

Reply:
fixed the logic in the component